### PR TITLE
Add sn07 playbook and other related things

### PR DESCRIPTION
**Pull request by sanjaysrikakulam**
Made changes related to sn07 and RHEL 9.

Attach this PR to: https://github.com/usegalaxy-eu/issues/issues/352

**Notes and changes:**
1. Created [DNS entry](https://github.com/usegalaxy-eu/infrastructure/pull/129)
2. Added `sn07.galaxyproject.eu` to the ansible inventory file
3. Added new requirements to the `requirements.yml` file
4. Added group_vars/sn07.yml file and made the following changes (compared to `sn06.yml`)
	1. Miniconda role variables (Python 3.9 is the default system version in RHEL 9, we need Python 3.8 so we will use Miniconda environment and install Python 3.8, pip and virtualenv packages and use them instead of the default system packages.
	2. [HTCondor does not support RHEL 9 yet](https://github.com/usegalaxy-eu/issues/issues/353), so we have set the variable `galaxy_systemd_handlers` to `0`. For the time being this second head node does not serve any job handlers.
	3. Made necessary condor config changes (variables: `condor_allow_negotiator`, and `condor_network_interface`) for the future.
	4. Updated the variable `galaxy_virtualenv_command` to use the virtualenv from the Miniconda `_galaxy_` environment we will create using the Miniconda role.
5. Added `sn07.yml` playbook and made the following changes (compared to `sn06.yml` playbook)
	1. variable `hostname` (obviously)
	2. Install dependencies in `pre_tasks`
	3. Included the Miniconda role and added required variables
	4. HTCondor role `usegalaxy_eu.htcondor` is commented out (see above for the reason)
	5. Rsync to NFS role `usegalaxy-eu.rsync-to-nfs` is commented out. We do not want to sync the Galaxy codebase from the host `sn07` for now. This can be changed when the head node `sn06` is also migrated to Rocky Linux 9.
	6. Included a new role `usegalaxy-eu.nginx-upload-module` for dynamically compiling the Nginx upload module and installing it.
5. Changes related to RHEL 9 were made to the following:
	1. Role: `ansible-collection-handy` (see PRs: https://github.com/usegalaxy-eu/ansible-collection-handy/pull/6, https://github.com/usegalaxy-eu/ansible-collection-handy/pull/7)
	2. @mira-miracoli made the following changes:
		1. Removed UWSGI monitoring stuff (`roles/hxr.monitor-uwsgi/tasks/main.ymll`, and `roles/hxr.monitor-uwsgi/templates/telegraf-uwsgi.sh`)
		2. Added workaround for including task `network_disks_access_time.yml` in `roles/usegalaxy-eu.monitoring/tasks/main.yml`. This task uses a tool called `pdsh` which is not available in RHEL 9. So this workaround will block the tasks from running on RHEL 9 systems.

### CODE REVIEW

File: hosts [hunk 1]
```diff
@@ -59,10 +59,14 @@ celery-0.galaxyproject.eu
 # Baremetal
 [galaxyservers]
 sn06.galaxyproject.eu ansible_ssh_user=centos
+sn07.galaxyproject.eu ansible_ssh_user=groot
```
Comment (bgruening) [inline hosts:62]: What is this user about? This user needs sudo access?

File: roles/usegalaxy-eu.monitoring/tasks/main.yml [hunk 2]
```diff
@@ -1,3 +1,6 @@
 ---
 - include_tasks: nfsstat.yml
+
+# Because pdsh is only available on RHEL8 and deprecated at RHEL9
```
Comment (bgruening) [inline roles/usegalaxy-eu.monitoring/tasks/main.yml:4]: Oh no its was so useful. Is there a replacement for the functionality?

File: sn07.yml [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+        - "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIFH54+qZEBeU5uwIeWWOViLcC509qxoRW6oN0VHRQr4r nate@treehouse"
+        - "ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBOBINXdjILF6x3WuppXyq6J2a2oSLR6waZ6txgjYJogHdIKPbI0TdReCv4EVxxYRY/NqGpHbjkqfRTsf2VgoU3U= mk@galaxy-mira"
+        - https://github.com/wm75.keys
+        - https://github.com/gmauro.keys
+    - name: Set authorized SSH key (stats user)
+      ansible.posix.authorized_key:
+        user: "stats"
+        state: present
+        key: "{{ item }}"
+      loop:
```
Comment (bgruening) [inline sn07.yml:106]: This brings up an interesting question, how do we provide access to those people when we are in an intern network? Not supporting this is an option.

File: hosts (hunk 1 above)
Comment (sanjaysrikakulam) [reply hosts:62]: That was the local user we created on the host while installing the server. The user can be removed and replaced with a `centos` user like everywhere else (Based on the other hosts, I assume that the `centos` user is the local admin user account?). The user is already given Sudo privileges while installing the host.

Comment (bgruening) [reply hosts:62]: Yes centos is the local admin account. At some point we should change to rocky (or what ever the new name is), but I would not introduce groot now, but change it either to centos or rocky. Does this make sense?

**Comment by bgruening** [comment]
As a first step I would also disable the workflow-schedulers. So we can test the web-handlers and if we are happy we enable the workflow-schedulers.

**Comment by sanjaysrikakulam** [comment]
Sure, I will push a new commit (will set the variable `galaxy_systemd_workflow_schedulers` to `0`).

**Review by mira-miracoli — APPROVED** [review]
Looks good to me, we could remove the ansible_ssh_user=centos from hosts, because it is allready in default group, but its not super important.
Thank you Sanjay!

**Pull request merged**